ansible_collections/arista/avd/molecule/eos_designs_negative_unit_tests/inventory/group_vars/SPINES.yml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Since there is only one switch leveraging these vars, just move it to that switch' hostvars file. Also change the hostname to follow the same style with duplicate....

Reply:
moved and changed the name to `duplicate-ip-address-uplink-switch-router-bgp`